backend/api/Areas/Property/Mapping/Parcel/BuildingParcelMap.cs

```diff
@@ -0,0 +1,105 @@
+        private readonly JsonSerializerOptions _serializerOptions;
+        #endregion
+
+        #region Constructors
+        /// <summary>
+        /// Creates a new instance of a BuildingParcelMap, initializes with specified arguments.
+        /// </summary>
+        /// <param name="serializerOptions"></param>
+        public BuildingParcelMap(IOptions<JsonSerializerOptions> serializerOptions)
+        {
+            _serializerOptions = serializerOptions.Value;
+        }
+        #endregion
+
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.ParcelBuilding, Model.BuildingParcelModel>()
+                .EnableNonPublicMembers(true)
+                .Map(dest => dest.Id, src => src.Parcel.Id)
+                .Map(dest => dest.PID, src => src.Parcel.ParcelIdentity)
+                .Map(dest => dest.PIN, src => src.Parcel.PIN)
+                .Map(dest => dest.ProjectNumber, src => src.Parcel.ProjectNumber)
+                .Map(dest => dest.Name, src => src.Parcel.Name)
+                .Map(dest => dest.Description, src => src.Parcel.Description)
+                .Map(dest => dest.ClassificationId, src => src.Parcel.ClassificationId)
+                .Map(dest => dest.Classification, src => src.Parcel.Classification.Name)
+                .Map(dest => dest.AgencyId, src => src.Parcel.AgencyId)
+                .Map(dest => dest.Agency, src => AgencyConverter.ConvertAgency(src.Parcel.Agency))
+                .Map(dest => dest.SubAgency, src => AgencyConverter.ConvertSubAgency(src.Parcel.Agency))
+                .Map(dest => dest.Latitude, src => src.Parcel.Location.Y)
+                .Map(dest => dest.Longitude, src => src.Parcel.Location.X)
+                .Map(dest => dest.Address, src => src.Parcel.Address)
+                .Map(dest => dest.LandArea, src => src.Parcel.LandArea)
+                .Map(dest => dest.LandLegalDescription, src => src.Parcel.LandLegalDescription)
+                .Map(dest => dest.Zoning, src => src.Parcel.Zoning)
+                .Map(dest => dest.ZoningPotential, src => src.Parcel.ZoningPotential)
+                .Map(dest => dest.IsSensitive, src => src.Parcel.IsSensitive)
+                .Map(dest => dest.IsVisibleToOtherAgencies, src => src.Parcel.IsVisibleToOtherAgencies)
+                .Map(dest => dest.Buildings, src => src.Parcel.Buildings)
+                .Map(dest => dest.Evaluations, src => src.Parcel.Evaluations)
+                .Map(dest => dest.Fiscals, src => src.Parcel.Fiscals)
+                .AfterMapping((src, dest) =>
+                {
+                    var metadata = JsonSerializer.Deserialize<Entity.Models.LeasedLandMetadata>(src.LeasedLandMetadata ?? "{}", _serializerOptions);
+
+                    dest.OwnershipNote = metadata.OwnershipNote;
+                })
+                .Inherits<Entity.BaseEntity, BModel.BaseModel>();
+
+            config.NewConfig<Model.BuildingParcelModel, Entity.Parcel>()
```

Comment:
map the flat FE building -> parcel relationship to the join table relationship of the backend.